File: qlty-cli/tests/lang/python/basic.in/identical.py
def f0(numbers):
    mean = sum(numbers) / len(numbers)

    sorted_numbers = sorted(numbers)
    if len(sorted_numbers) % 2 == 0:
        mid_index = len(sorted_numbers) // 2
        median = (sorted_numbers[mid_index - 1] + sorted_numbers[mid_index]) / 2
    else:
        median = sorted_numbers[len(sorted_numbers) // 2]

    return mean, median

def f1(numbers):
    mean = sum(numbers) / len(numbers)

    sorted_numbers = sorted(numbers)
    if len(sorted_numbers) % 2 == 0:
        mid_index = len(sorted_numbers) // 2
        median = (sorted_numbers[mid_index - 1] + sorted_numbers[mid_index]) / 2
    else:
        median = sorted_numbers[len(sorted_numbers) // 2]

    return mean, median

File: qlty-cli/tests/lang/kotlin/basic.in/identical.kt
fun f0(numbers: IntArray): Pair<Double, Int?> {
    val mean: Double = numbers.sum().toDouble() / numbers.size.toDouble()

    val sortedNumbers = numbers.sorted()
    val median: Int? = if (sortedNumbers.size % 2 == 0) {
        val midIndex = sortedNumbers.size / 2
        (sortedNumbers[midIndex - 1] + sortedNumbers[midIndex]) / 2
    } else {
        sortedNumbers[sortedNumbers.size / 2]
    }

    return Pair(mean, median)
}

fun f1(numbers: IntArray): Pair<Double, Int?> {
    val mean: Double = numbers.sum().toDouble() / numbers.size.toDouble()

    val sortedNumbers = numbers.sorted()
    val median: Int? = if (sortedNumbers.size % 2 == 0) {
        val midIndex = sortedNumbers.size / 2
        (sortedNumbers[midIndex - 1] + sortedNumbers[midIndex]) / 2
    } else {
        sortedNumbers[sortedNumbers.size / 2]
    }

    return Pair(mean, median)
}
